routers/todos.py

```diff
@@ -0,0 +1,122 @@
+from database_config import local_session
+from fastapi import APIRouter, Depends, HTTPException, Path
+from models import Todo
+from pydantic import BaseModel, Field
+from sqlalchemy import and_
+from starlette import status
+
+from .auth import get_current_user, get_db
+
+todo_router = APIRouter(
+    prefix="/todos", tags=["Todos"], responses={401: {"user": "user_not_authenticated"}}
+)
+
+
+class TodoModel(BaseModel):
+    title: str = Field(min_length=0, max_length=32)
+    discription: str = Field(min_length=0, max_length=100)
+    priority: int = Field(gt=-1, lt=6)
+
+    class Config:
+        schema_extra = {
+            "example": {
+                "title": "Title of the todo",
+                "discription": "Some detail about todo",
+                "priority": 3,
+            }
+        }
+
+
+@todo_router.get("/", status_code=status.HTTP_200_OK)
+async def get_all_todos(
+    db: local_session = Depends(get_db), user: dict = Depends(get_current_user)
+) -> list:
```

Comment:
I mentioned this in a previous PR but when returning iterables, it's good practice to mention the types of the elements in the list like so
```suggestion
) -> list[Todo]:
```